MogoPocTest/LocationDbFixture.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using Bks.DataAccess.Mongo;
+using FakeItEasy;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using MongoDB.Driver;
+using MongoPocWebApplication1.Infrastructure.Mongo;
+
+namespace MogoPocTest
+{
+    public class LocationDbFixture : IDisposable
```

Comment:
As we discussed, you never call DB in integration tests.

All presets (create items or update their properties) are executing via service API. All validations should be done also via API.

For instance, you want to test Delete endpoint:
1. Call POST to create a new item. It returns 200 with a new Id.
2. Call Delete /id should return 200.
3. Call to Get /id should return 404.

Reply:
How would you add regression test for the repository Query / CRUD logic ?